nixos/modules/services/misc/paperless.nix

```
@@ -571,6 +585,15 @@ in
             gid = config.ids.gids.paperless;
           };
         };
+
+        services.gotenberg = lib.mkIf cfg.configureTika {
+          enable = true;
```

```suggestion
          enable = true;
          chromium.disableJavascript = true;
          extraArgs = "--chromium-allow-list=file:///tmp/.*";
```
maybe disable javascript and allow only local? its recommended by the [docker example in paperless](https://github.com/paperless-ngx/paperless-ngx/blob/main/docker/compose/docker-compose.sqlite-tika.yml#L64)

---

Thank you for letting me know about these recommendations, they are now included in this PR.